app/templates/core/create-account-modal/choose-account-type-view.jade

```diff
@@ -1,56 +1,40 @@
+
 .modal-body-content
   if me.get("preferredLanguage") == "he" || window.location.host == "il.codecombat.com"
     .alert.alert-info.text-center(dir="rtl")
       span תלמיד מערכת החינוך?
       a(href="https://pub.skillz-edu.org/portal/playground")  התחבר דרך פורטל משרד החינוך
 
-  h4.choose-type-title
-    span(data-i18n="signup.choose_type")
+
   .path-cards
-    .path-card.navy
-      .card-title
-        span(data-i18n="courses.teacher")
-      .card-content
-        h6.card-description
-          span(data-i18n="signup.teacher_type_1")
-        ul.small.m-t-1
-          li
-            span(data-i18n="signup.teacher_type_2")
-          li
-            span(data-i18n="signup.teacher_type_3")
-          li
-            span(data-i18n="signup.teacher_type_4")
+    .path-card
+      img.card-image(alt='Educator' src='/images/pages/account/create/educator.png')
       .card-footer
-        button.btn.btn-lg.btn-navy.teacher-path-button
-          .text-h6
-            span(data-i18n="signup.signup_as_teacher")
+        i.small
+          div(data-i18n="signup.im_a_teacher_description") Teach Computer Science with CodeCombat
+        button.primary-oz-button.teacher-path-button
+          span(data-i18n="signup.im_a_teacher_button") I'm a Teacher
 
-    .path-card.forest
-      .card-title
-        span(data-i18n="courses.student")
-      .card-content
-        h6.card-description
-          span(data-i18n="signup.student_type_1")
-        ul.small.m-t-1
-          li
-            span(data-i18n="signup.student_type_2")
-          li
-            span(data-i18n="signup.student_type_3")
-          li
-            span(data-i18n="signup.student_type_4")
+    .path-card
+      img.card-image(alt='Educator' src='/images/pages/account/create/parent.png')
+      .card-footer
+        i.small
+          div(data-i18n="signup.im_a_parent_description") Explore our live online coding classes
+        button.primary-oz-button.parent-path-button
+          span(data-i18n="signup.im_a_parent_button") I'm a Parent
+  .path-cards
+    .path-card
+      img.card-image(alt='Educator' src='/images/pages/account/create/student.png')
       .card-footer
         i.small
-          span(data-i18n="signup.student_type_5")
-        button.btn.btn-lg.btn-forest.student-path-button
-          .text-h6
-            span(data-i18n="signup.signup_as_student")
+          div Learn Computer Science with your class
+        button.primary-oz-button.student-path-button
+          span(data-i18n="signup.im_a_student_button") I'm a Student
 
-  if me.showIndividualRegister()
```

Comment:
I think we still need `showIndividualRegister()` checks to avoid showing parent or individual account creation options on our China server, but @smallst may be able to confirm.

Reply:
@smallst can you help here?

Reply:
@adamkecskes can u add that check for now?

Reply:
@mrfinch the check added, but also restructured a bit, because now we have to hide both parent and individual.